src/main/java/com/kodilla/ecommercee/service/UserDbService.java

```diff
@@ -0,0 +1,64 @@
+package com.kodilla.ecommercee.service;
+
+import com.kodilla.ecommercee.dao.ProductsGroupDao;
+import com.kodilla.ecommercee.dao.UserDao;
+import com.kodilla.ecommercee.domain.ProductsGroup;
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.exception.UserNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class UserDbService {
```

Comment:
Tu przed zmergowaniem byłoby dobrze usunąć niepotrzebny zakomentowany kod :)

Reply:
Dzieki jakos to przeoczylem. Consider done.